python/golem_task_api/client.py

```diff
@@ -78,11 +78,15 @@ def __init__(
     async def next_subtask(
             self,
             task_id: str,
-    ) -> Tuple[str, dict]:
+    ) -> Tuple[str, dict, List[str]]:
```

Comment:
Maybe introduce a simple data structure like this? (because tuples are ugly)
```python
class Subtask(NamedTuple):
    subtask_id: str
    params: dict
    resources: List[str]
```